ddtrace/contrib/psycopg/__init__.py

```diff
@@ -60,9 +60,18 @@
     cursor = db.cursor()
     cursor.execute("select * from users where id = 1")
 """
-from .patch import get_version
-from .patch import get_versions
-from .patch import patch
+from ...internal.utils.importlib import require_modules
 
 
-__all__ = ["patch", "get_version", "get_versions"]
+required_modules = ["psycopg", "psycopg2"]
+with require_modules(required_modules) as missing_modules:
+    if not missing_modules:
```

Comment:
Either `psycopg` (psycopg3) or `psycopg2` will be installed. It highly unlikely that BOTH will installed. Therefore `missing_modules` will usually contain one element. In this scenario `not missing_modules`  will evaluate to False and patch will not be imported. This explains the failures we are currently seeing in CI.